hpack/hpack.py

```diff
@@ -295,12 +295,16 @@ def header_table_size(self):
     def header_table_size(self, value):
         self.header_table.maxsize = value
 
-    def decode(self, data):
+    def decode(self, data, raw=False):
         """
         Takes an HPACK-encoded header block and decodes it into a header set.
 
         :param data: A bytestring representing a complete HPACK-encoded header
                      block.
+        :param raw: An optional boolean that when True specifies that raw byte
+                    strings for the name and value should be returned instead
+                    of automatically decoding them as UTF-8 values. The default
+                    value is False.
```

Comment:
Let's reword this slightly, it's a little tricky to follow.

Something like: "(optional) Whether to return the headers as tuples of raw byte strings or to decode them as UTF-8 before returning them. The default value is False, which returns tuples of Unicode strings."